Software/scan_key/AuthenticationSQL.py

```diff
@@ -1,42 +1,30 @@
 from on_off import *
 from ErrorSQL import *
 import os
+import subprocess
 
 def machineAuth(id, cursor):
-    os.system("hostname > tmp")
-    currMachineID = open('tmp', 'r').read()
-    currMachineID = currMachineID.split()
+    currMachineID = subprocess.check_output(['hostname'])
+    currMachineID = currMachineID.strip()
+    currMachineID = b(currMachineID).decode('UTF-8')
 
-    cursor.execute("select machineID, machineType from MACHINE")
+    cursor.execute("SELECT machineType FROM MACHINE WHERE machineID = " + currMachineID)
+    
+    machineType = cursor.fetchone()
 
-    data1 = cursor.fetchall()
+    cursor.execute("SELECT auth1, auth2, auth3 FROM USER WHERE t1String = " + id) #getting user w/ the id    #might need to modify
 
-    machineType = None
+    authdata = cursor.fetchone()
```

Comment:
We don't want to download all three cols - just the one that is appropriate for our machine type.